src/main/java/com/xceptance/neodymium/NeodymiumRunner.java

```diff
@@ -200,9 +200,15 @@ protected List<FrameworkMethod> computeTestMethods()
         // from methodBlock
         List<Class<? extends StatementBuilder>> statementRunOrder = new DefaultStatementRunOrder().getRunOrder();
 
+        List<FrameworkMethod> computedMethods = super.computeTestMethods();
+        boolean wipMethod = computedMethods.stream().anyMatch(computedMethod -> computedMethod.getAnnotation(WIP.class) != null);
         // super.computeTestMethods will return all methods that are annotated with @Test
```

Comment:
Comment describes a line of code that is not below it any more, but on line 203 now. Needs to get shifted upwards a bit

Reply:
done